packages/wallet/ui/src/components/Offer.jsx

```diff
@@ -147,6 +149,14 @@ const OfferWithoutContext = ({
       <ErrorBoundary>
         <Proposal offer={offer} />
       </ErrorBoundary>
+      {signingError && (
+        <div className="OfferEntry">
+          <h6>Signing Error</h6>
+          <details style={{ whiteSpace: 'pre-wrap' }}>
+            {`${signingError}`}
```

Comment:
I don't know if we have style guidance on this but this looks like a cast to me,
```suggestion
            {String(signingError)}
```
the string template makes me search for what's in the template and the cast makes clear more quickly what the intent is.

Reply:
Sounds good, never thought of doing `String()`